routes/user.js

```diff
@@ -3,6 +3,12 @@ var router = express.Router();
 var passport = require('passport');
 var User = require('../models/user');
 var Playlist = require('../models/playlist')
+var app = require('../app')
+var pdf = require('html-pdf')
+var fs = require('file-system')
+var Song = require('../models/song')
+var ExportSong = require('../models/exportSong')
+var _und = require('../bower_components/underscore/underscore-min')
```

Comment:
i recommend lodash. Also if you do npm install --save lodash, then here you can do 
`var _ = require('lodash')`